tests/BehatExtensionTest.php

```diff
@@ -9,78 +9,53 @@
 
 use function file_get_contents;
 use function sys_get_temp_dir;
+use function unlink;
 
 class BehatExtensionTest extends TestCase
 {
-    public function testPrintsUnusedStepDefinitions(): void
+    /**
+     * @return array<array{0: string}>
+     */
+    public static function behatProfilesProvider(): array
     {
-        $behat = new Process(['../../vendor/bin/behat', '--config', 'behat.yml'], __DIR__ . '/fixtures/');
-        $behat->mustRun();
-
-        $this->assertStringContainsString('2 unused step definitions:', $behat->getOutput());
-
-        $this->assertStringContainsString(
-            'Given some precondition that is never used in a feature # FeatureContext::somePrecondition()',
-            $behat->getOutput()
-        );
-        $this->assertStringContainsString(
-            'Then some step that is never used by a feature # FeatureContext::someStepThatIsNeverUsedByAFeature()',
-            $behat->getOutput()
-        );
+        return [
```

Comment:
Because of the use of the data provider, the (testdox) output of PHPUnit has become a little bit meaningless:

![afbeelding](https://github.com/nicwortel/behat-unused-step-definitions-extension/assets/1055691/e1f6e105-87f1-4241-8e99-6746a493ff8c)

This makes the debugging a bit complex if any of those tests would break at some point in the future. Because of that I tend to avoid using data providers, except in cases where I'm actually using multiple examples to test the same logic path.